packages/common/src/services/remote-config/defaults.ts

```diff
@@ -4,6 +4,11 @@ const ETH_PROVIDER_URLS = process.env.REACT_APP_ETH_PROVIDER_URL || ''
 const DEFAULT_ENTRY_TTL = 1 /* min */ * 60 /* seconds */ * 1000 /* ms */
 const DEFAULT_HANDLE_VERIFICATION_TIMEOUT_MILLIS = 5_000
 
+export const MIN_USDC_PURCHASE_AMOUNT_CENTS = 100
+export const MAX_USDC_PURCHASE_AMOUNT_CENTS = 150000
+export const MIN_CONTENT_PRICE_CENTS = 100
+export const MAX_CONTENT_PRICE_CENTS = 150000
```

Comment:
This is also a little weird. The way the typing is defined for `getRemoteVar` and `remoteConfig(type)Defaults` means that you _always_ have to null check values, even though I think they are guaranteed to be defined at runtime for almost all of our `IntKeys`

Reply:
agreed